### DISCUSSION TITLE
Israel Channel

### DISCUSSION THREAD

**Discussion by adinaveh** [iptv-org/community#430] [Show and tell]
Shelanu (IL) - https://1247634592.rsc.cdn77.org/1247634592/tracks-v1a1/mono.m3u8

**Comment by MichaelCaditz** [comment]
That is not an Israel channel. It is an American Christian evangelist broadcast which was thrown off the air by the Israel government for deception.

https://www.christianitytoday.com/news/2020/june/israel-orders-shelanu-god-tv-hebrew-christian-channel-off.html

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
